src/tests/passthrough.spec.ts

```diff
@@ -36,3 +37,85 @@ describe("Passthrough", () => {
     });
   });
 });
+
+describe("Passthrough with grpc-web+json unframing", () => {
+  setupServers({ mode: "passthrough", enableUnframeGrpcWebJson: true });
+
+  test("response: simple text", async () => {
+    const response = await axios.get(`${PROXAY_HOST}${SIMPLE_TEXT_PATH}`);
+    expect(response.data).toBe(SIMPLE_TEXT_RESPONSE);
+  });
+
+  test("response: utf-8", async () => {
+    const response = await axios.get(`${PROXAY_HOST}${UTF8_PATH}`);
+    expect(response.data).toBe(UTF8_RESPONSE);
+  });
+
+  test("response: binary", async () => {
+    const response = await axios.get(`${PROXAY_HOST}${BINARY_PATH}`, {
+      responseType: "arraybuffer",
+    });
+    expect(response.data).toEqual(BINARY_RESPONSE);
+  });
+
+  test("can pick any tape", async () => {
+    await axios.post(`${PROXAY_HOST}/__proxay/tape`, {
+      tape: "new-tape",
+    });
+  });
```

Comment:
These are all copies of the above tests, but with the proxay server running with unframing enabled. I duplicated these to ensure they weren't affected by running with unframing enabled.